src/amt/DeviceAction.ts

```diff
@@ -392,4 +392,12 @@ export class DeviceAction {
     logger.silly(`deleteAlarmClockOccurrences ${messages.COMPLETE}`)
     return deleteResponse.Envelope
   }
+
+  async unprovisionDevice (): Promise<Common.Models.Envelope<any>> {
```

Comment:
@Walt-Intel Can you add the following test to the deviceAction.test.ts
`describe('unprovision amt', () => {
    it('should return response from AMT', async () => {
      sendSpy.mockResolvedValueOnce({Envelope:null})
      const result = await device.unprovisionDevice()
      expect(result).toBe(null)
    })
  })`